docs/GCSBucketCreate-action.md

```diff
@@ -1,3 +1,40 @@
-## Google Cloud Storage Create
+# Google Cloud Storage Create
 
-This plugin is used for creating directories on Google Cloud Storage (GCS).
+Description
+-----------
+This plugin creates objects in a Google Cloud Storage bucket.
+Cloud Storage allows world-wide storage and retrieval of any amount of data at any time.
+
+Buckets are the basic containers that hold your data.
+Everything that is stored in Cloud Storage must be contained in a bucket.
+Buckets are used to organize and control access to data.
+
+Objects are the individual pieces of data that are stored in Cloud Storage.
+Object names can contain any combination of Unicode characters (UTF-8 encoded) and must be less than 1024 bytes in length.
+Object names often look like file paths.
+
+Authorization
+-------------
+If the plugin is run on a Google Cloud Dataproc cluster, the service account key does not need to be provided.
+Credentials will be automatically read from the cluster environment.
+
+If the plugin is not run on a Dataproc cluster, the path to a service account key must be provided.
+The service account key can be found on the Dashboard in the Cloud Platform Console.
+Make sure the account key has permission to access BigQuery and Google Cloud Storage.
+The service account key file needs to be available on every node in your cluster and
+must be readable by all users running the job.
+
+Properties
+----------
+**Project ID**: The Google Cloud Project ID, which uniquely identifies a project.
+It can be found on the Dashboard in the Google Cloud Platform Console.
+
+**Service Account File Path**: Path on the local file system of the service account key used for
+authorization. Does not need to be specified when running on a Dataproc cluster.
+When running on other clusters, the file must be present on every node in the cluster.
+
+**Bucket Name**: The bucket to create objects in.
+
+**Objects to Create**: The objects to create in the bucket.
```

Comment:
will be good to give an example for this as the backend expects it in comma separated format.

Reply:
For our reference docs, we're writing all new ones from the perspective of a UI user, who doesn't know anything about the underlying format.